src/app/providers.tsx

```diff
@@ -23,7 +23,7 @@ export function Providers({ children }: { children: React.ReactNode }) {
     id: process.env.NEXT_PUBLIC_GOOGLE_TAG_MANAGER_CONTAINER_ID ?? '',
   }
   return (
-    <CacheProvider>
+    <CacheProvider prepend={true}>
```

Comment:
With this there should be less issues between chakra <> tailwind - as chakra should be loaded first and tailwind styles are applied afterwards.